src/Apps/Legion/Components/WizardShell/Internal/ModifyStep/Internal/DiagramTab/DiagramTab.tsx

```diff
@@ -0,0 +1,137 @@
+import React, {
+    useCallback,
+    useEffect,
+    useMemo,
+    useRef,
+    useState
+} from 'react';
+import { getDebugLogger } from '../../../../../../../../Models/Services/Utils';
+import { IDiagramTabProps } from './DiagramTab.types';
+import { useClassNames } from './DiagramTab.styles';
+import { useEntities } from '../../../../../../Hooks/useEntities';
+import { Kind } from '../../../../../../Models';
+import Diagram from '../../../../../Diagram/Diagram';
+import { TDiagramAnnotation } from '../../../../../Diagram/Diagram.types';
+import {
+    PIDSourceUrls,
+    PIDSourceUrlsToImgUrlMapping,
+    PID_EXTRACTED_PROPERTIES
+} from '../../../../../../Models/Constants';
+import { Dropdown, IDropdownOption } from '@fluentui/react';
```

Comment:
Dropdown is available in Fluent 9, could be interesting to see if it works here.